src/index.js

```diff
@@ -68,6 +68,23 @@ function runAsync (command, options) {
   })
 }
 
+function runAsyncSpawn (command, args, options) {
+  return new Promise((resolve, reject) => {
+    const spawnOptions = Object.assign({shell: true}, options)
+    const asyncProcess = childProcess.spawn(command, args, spawnOptions)
+    asyncProcess.on('close', code => {
+      if (code !== 0) {
+        reject('failed')
+      } else {
+        resolve('success')
+      }
+    })
+    asyncProcess.on('error', (error) => {
```

Comment:
As far as I can tell it means the command failed to start. I wasn't able to make that happen though.